config/doctrine/LineItem.orm.xml

```diff
@@ -18,5 +18,6 @@
                 <join-column name="infrastructure_id" referenced-column-name="id"/>
             </join-columns>
         </many-to-one>
+        <field name="maxAttempts" type="integer" column="max_attempts" precision="0" scale="0" nullable="true"/>
```

Comment:
How do you think, maybe it would be better to have default = 0?
Smth like:
```xml

        <field name="maxAttempts" type="integer" column="max_attempts" nullable="false">
            <options>
                <option name="default">0</option>
            </options>
        </field>
```